src/main/java/io/antmedia/AntMediaApplicationAdapter.java

```diff
@@ -546,13 +546,73 @@ public void resetDASHStats(String streamId) {
 		}
 	}
 
+	public void sendStartPlayWebHook(final String streamId, final String viewerId, final String token){
+		final Broadcast broadcast = getDataStore().get(streamId);
+		if(broadcast == null){
```

Comment:
There is a code repetition in the new methods. Following lines are same
```
final String listenerHookURL = broadcast.getListenerHookURL();
		if (listenerHookURL == null || listenerHookURL.isEmpty()) {
			return;
		}
		final String name = broadcast.getName();
		final String category = broadcast.getCategory();
		final String metaData = broadcast.getMetaData();
		logger.info("Setting timer to call viewer play started hook for stream:{}", streamId);
		vertx.setTimer
```

Create a method and call that method with the proper parameter

Reply:
done